src/Components/Modal/ModalDialog.tsx

```diff
@@ -0,0 +1,89 @@
+import { color, Flex, Sans } from "@artsy/palette"
```

Comment:
All new components (for the time being) should go in `Styleguide/Components`. That's the current boundary for before / after our design system implementation. At a certain point we'll find a proper way to organize but that should happen in a refactor pass rather than within a feature.

Reply:
Ah, I see - this has been grouped with other existing modal stuff.